app/uk/gov/hmrc/claimvatenrolmentfrontend/forms/CaptureBox5FigureForm.scala

```diff
@@ -51,6 +51,11 @@ object CaptureBox5FigureForm {
   )
 
   val form: Form[String] = Form(
-      box5Figure -> text.verifying(box5FigureEmpty andThen box5FigureLength andThen box5FigureFormat)
+    single(
+      box5Figure -> text
+        .transform[String](_.replaceAll(",",""), identity)
```

Comment:
why do we strip commas here?

Reply:
got it, from jira, thanks
"NOTE: need to make sure we will allow, but then remove commas from any inputs. e.g. if user enters 12,123,123.45 we accept it but strip out allow the commas on what we store so that the values stored is 12123123.45"